### PR TITLE
add 1.3 and strict DI

### PR DESCRIPTION
**Pull request by 0xF013**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-feature-angular-1-3-sctrict-di.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by 0xF013** [comment]
https://www.npmjs.org/package/ng-annotate-webpack-plugin

File: gulpfile.js [hunk 1]
```diff
@@ -59,6 +62,24 @@ gulp.task('webpack', function() {
...
 
+gulp.task('minify', function() {
+  return gulp.src('frontend/app/openproject-app.js')
+    .pipe(gulpWebpack(config))
+    .pipe(ngAnnotate())
+    .pipe(uglify())
+    .pipe(gulp.dest('app/assets/javascripts/bundles'));
+});
+
+gulp.task('templates', function () {
```
Comment (myabc) [inline gulpfile.js:73]: @0xF013 FYI, this won't work with plugins.

**Comment by myabc** [comment]
I've split this PR and rebased - see #2279 for Angular 1.3 upgrade.

**Comment by hschink** [comment]
@0xF013 @myabc Do we want to close this PR?

**Comment by 0xF013** [comment]
@hschink @myabc ok for me

**Comment by myabc** [comment]
ja

**Pull request closed without merging**